backend/database/SQL script/init.sql

```diff
@@ -63,37 +70,37 @@ CREATE TABLE Customers (
     PRIMARY KEY (userId),
 	FOREIGN KEY (userId) REFERENCES Users
 		on DELETE CASCADE
+		on UPDATE CASCADE
 );
 
 CREATE TABLE Riders (
 	userId 		INTEGER,
 	area 		VARCHAR(20) NOT NULL,
-	salary		NUMERIC(8,2),	
```

Comment:
confused about why the salary attribute is removed. Where do we store the rider's salary now ah?